Patch the _is_conv_node function

cccclai opened the pull request:
Summary: torch.ops.aten.conv2d.padding is also conv2d node

Differential Revision: D74898941

Skylion007 commented on torch/ao/quantization/pt2e/utils.py:
```diff
@@ -168,6 +168,7 @@ def _is_conv_node(n: Node):
     return n.op == "call_function" and n.target in [
         torch.ops.aten.conv1d.default,
         torch.ops.aten.conv2d.default,
+        torch.ops.aten.conv2d.padding,
```
```suggestion
        torch.ops.aten.conv1d.default,
        torch.ops.aten.conv1d.padding,
        torch.ops.aten.conv2d.default,
        torch.ops.aten.conv2d.padding,
```
What about conv3d?

jerryzh168 replied:
I added conv3d,

@cccclai might be good to abandon these changes

Skylion007 replied:
Wait so this func is unused now?

cccclai replied:
Our partner is still using pytorch but not torch ao currently, I think it's an easy fix?

Skylion007 replied:
@cccclai Might as well be complete and ad all the conv types then.

Skylion007 commented:
Do we need to add any tests for the quantization ops?

Skylion007 reviewed (approved):
LGTM if you also fix the utility here

cccclai commented:
> LGTM if you also fix the utility here

Does it look good now?

Skylion007 commented:
@cccclai Yeah, just fix the trailing spaces in the lint

cccclai commented:
I added the test in https://github.com/pytorch/ao/pull/2223. The same test doesn't work in pytorch, looks like the quant nodes disappear.

cccclai committed:
Patch the _is_conv_node function (#153749)

Summary:
X-link: https://github.com/pytorch/ao/pull/2223


torch.ops.aten.conv2d.padding is also conv2d node

Test Plan: CI

Reviewed By: andrewor14, jerryzh168

Differential Revision: D74898941

cccclai commented:
Close as the PR is reverted
